src/openrct2/drawing/Drawing.h

```diff
@@ -108,7 +109,14 @@ struct rct_drawpixelinfo
     int16_t pitch{}; // note: this is actually (pitch - width)
     ZoomLevel zoom_level{};
 
+    uint8_t remX{};
```

Comment:
What does this variable hold? It isn't clear from the name or context, and there is no comment either.